<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3000</width>
    <height>1650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwid">
   <widget class="QFrame" name="back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>500</width>
      <height>701</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="ready">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>500</width>
      <height>701</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="processingFrame">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>250</y>
      <width>651</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="nowProcessingLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;正在执行&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suspendButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>710</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>挂起</string>
    </property>
   </widget>
   <widget class="QPushButton" name="releaseSusButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>1450</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>解挂</string>
    </property>
   </widget>
   <widget class="QFrame" name="addframe">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>770</y>
      <width>651</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>1340</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>1340</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>1340</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>继续</string>
    </property>
   </widget>
   <widget class="QPushButton" name="revertButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>1340</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>还原</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suspendProcessingButton">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>710</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>挂起</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>2400</x>
      <y>260</y>
      <width>431</width>
      <height>1111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="detailLayout"/>
   </widget>
   <widget class="QPushButton" name="releaseAllSusButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>1450</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>全部解挂</string>
    </property>
   </widget>
   <widget class="QLabel" name="detailLabel">
    <property name="geometry">
     <rect>
      <x>2330</x>
      <y>210</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;详细信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="intervalspinBox">
    <property name="geometry">
     <rect>
      <x>2380</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="intervalLabel">
    <property name="geometry">
     <rect>
      <x>2230</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>观察延时</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ThreadsspinBox">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="threadsLabel">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>就绪队列大小</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CorespinBox">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CoreLabel">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU核心数</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>260</y>
      <width>371</width>
      <height>1141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="memLayout"/>
   </widget>
   <widget class="QFrame" name="suspend">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>770</y>
      <width>500</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="swaping">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>770</y>
      <width>500</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>160</y>
      <width>651</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="memProgressLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;总内存:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="allMemSize">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;使用中:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="memUsingSize">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;空闲:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="memLeftSize">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="fitAlgoCombo">
    <property name="geometry">
     <rect>
      <x>1830</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>最先适应算法</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最优适应算法</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>内存分配算法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>120</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>主存大小</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="allMemSizeEdit">
    <property name="geometry">
     <rect>
      <x>1830</x>
      <y>110</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>2010</x>
      <y>260</y>
      <width>391</width>
      <height>1141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="memDetailLayout"/>
   </widget>
   <widget class="QPushButton" name="allMemSizeBtn">
    <property name="geometry">
     <rect>
      <x>2030</x>
      <y>110</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
